src/common/imagpng.cpp

```diff
@@ -37,6 +41,8 @@
 
 // For memcpy
 #include <string.h>
+#include <map> // for custom option names
```

Comment:
We still support building wx on the systems without (decent implementation of) the standard library. Perhaps this is something we could change in 3.2, but until this is decided, we should continue to use our own stuff from `wx/hash{map,set}.h`.

Although in this particular case I wonder if we could just use a couple of `wxVector<>`s, it's not like we're going to have thousands of options, so a vector is probably going to outperform the map, in addition to being simpler to use.